### PR TITLE
Make minor fixes to Nexmark page

### PR DESCRIPTION
**Pull request by echauchot**

### CODE REVIEW

File: src/documentation/sdks/nexmark.md [hunk 1]
```diff
@@ -175,9 +175,9 @@ Available suites are:
     --workerMachineType=n1-highmem-8 \
     --stagingLocation=<a gs path for staging> \
     --runner=DataflowRunner \
-    --tempLocation=gs://talend-imejia/nexmark/temp/ \
-    --stagingLocation=gs://talend-imejia/nexmark/temp/staging/ \
-    --filesToStage=target/beam-sdks-java-nexmark-2.1.0-SNAPSHOT.jar
```
Comment (iemejia) [inline src/documentation/sdks/nexmark.md]: Better add {{ site.release_latest }} instead of -[version]  This applies to the other uses of version too.

Comment (echauchot) [reply src/documentation/sdks/nexmark.md]: yes, totally agree, good suggestion !

**Comment by echauchot** [comment]
done, PTAL

**Comment by iemejia** [comment]
Mmm can't test at this moment locally. There was supposed to be a staging link to test if this looks ok, no? What happened with this ? Maybe you know @melap ?

**Comment by echauchot** [comment]
rebased on asf-site

**Comment by iemejia** [comment]
@asfgit merge

**Comment by melap** [comment]
@iemejia I'm not sure why it isn't posting the message with the URL right now, but it is definitely still staging. you can see it at http://apache-beam-website-pull-requests.storage.googleapis.com/408/index.html  -- I just bookmark that link and change the PR # that's in the URL as a workaround in the meantime.

**Comment by iemejia** [comment]
Thanks for the ref @melap, at least I will know now how to check until it gets fixed. Do you know if there is a JIRA for this?

**Comment by melap** [comment]
@iemejia hmm, I thought there was but now I'm looking and can't find it. maybe I was thinking of the new file regeneration issue. I'll file a new JIRA for this one.

**Pull request merged**